templates/main.php

```diff
@@ -29,8 +29,8 @@
                             <span class="lot__amount">Стартовая цена</span>
                             <span class="lot__cost"><?= htmlspecialchars(format_price($lot['price'])); ?></span>
                         </div>
-                        <div class="lot__timer timer">
-                            12:23
+                        <div class="lot__timer timer<?php $exp_time = count_date($lot['expiration date']); echo ($exp_time['hours'] == 0) ? " timer--finishing" : " " ?>">
```

Comment:
В такой ситуации лучше объявление переменных вынести в другой php блок
```suggestion
                        <?php $exp_time = count_date($lot['expiration date']); ?>
                        <div class="lot__timer timer<?= ($exp_time['hours'] == 0) ? " timer--finishing" : ""; ?>">
```